Ginger/Ginger/Variables/VariablesPage.xaml.cs

```diff
@@ -344,8 +344,12 @@ private void GenerateVariablesValue(object sender, RoutedEventArgs e)
             if (grdVariables.Grid.SelectedItems.Count == 0)
                 Reporter.ToUser(eUserMsgKey.NoItemWasSelected);
             else
+            {
+                string errorMsg = string.Empty;
                 foreach (object var in grdVariables.Grid.SelectedItems)
-                    ((VariableBase)var).GenerateAutoValue();
+                    ((VariableBase)var).GenerateAutoValue(ref errorMsg);
```

Comment:
and here? in any page calling auto generate value need to decide if relevant to show error message to user